technologies/job/spark/spark-2.4/innerContexts/python/spark-2.4-python-2.7/build.gradle.kts

```diff
@@ -15,26 +15,3 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import com.bmuschko.gradle.docker.DockerRemoteApiPlugin
```

Comment:
Why deleted this part ?

Reply:
It was a bad commit. It is back to normal now.